File: Selenium/Activities/activity4.java
package Activities;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class activity4 {
    public static void main(String[] args) throws InterruptedException {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();

        driver.get("https://www.training-support.net/selenium/target-practice");
        System.out.println("Title of the page = " +driver.getTitle());
        String text = driver.findElement(By.xpath("//h3[@id= 'third-header']")).getText();
        System.out.println("Tile of the 3rd hearder : " +text);
        String header2 = driver.findElement(By.xpath("//h5[@class= 'ui green header']")).getCssValue("color");
        System.out.println("Tile of the 5rd header : " +header2);
        String voilet = driver.findElement(By.xpath("//button[text() = 'Violet']")).getAttribute("class");
        System.out.println(voilet);
        String greyclor = driver.findElement(By.xpath("//button[contains(@class ,'ui grey button')]")).getText();
        System.out.println(greyclor);
        driver.quit();
    }
}

File: Selenium/Activities/activity4.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    driver.get("https://www.training-support.net/selenium/target-practice")
    print("Title of the page = " ,driver.Title())
    text = driver.find_element(By.XPATH("//h3[@id= 'third-header']")).text
    print("Tile of the 3rd header : ",text)
    header2 = driver.find_element(By.xpath("//h5[@class= 'ui green header']")).value_of_css_property("color")
    print("Tile of the 5rd header : ",header2)
    voilet = driver.find_element(By.xpath("//button[text() = 'Violet']")).get_attribute("classr")
    print(voilet);
    greyclor = driver.findElement(By.xpath("//button[contains(@class ,'ui grey button')]")).getText();
    print(greyclor)
    driver.quit()











S